core/ui_mixins/quick_panel.py

```diff
@@ -113,20 +113,28 @@ def get_arguments(self, selected_action):
 
 
 def show_paginated_panel(items, on_done, flags=None, selected_index=None, on_highlight=None,
-                         limit=6000, next_message=None):
+                         limit=6000, format_item=None, next_message=None, status_message=None):
     """
     Display items in quick panel with pagination, and execute on_done
     when item is selected.
 
     items: can be either a list or a generator.
+
     on_done: a callback will take one argument
+
     limit: the number of items per page
+
     selected_index: an integer or a callable returning boolean.
                     If callable, takes either an integer or an entry.
+
     on_highlight: a callable, takes either an integer or an entry.
 
+    format_item: a function to format each item
+
     next_message: a message of next page, default is ">>> NEXT PAGE >>>"
 
+    status_message: a message to display at statusbar while loading the entries.
```

Comment:
How do you feel about moving this docstring to the `PaginatedPanel` class below instead, separating the class' static properties from overwritable methods to make it clear?